data_model/src/permission/token.rs

```diff
@@ -230,8 +155,31 @@ impl Registered for Definition {
 }
 
 impl Definition {
-    /// Construct new `PermissionTokenDefinition`
-    pub fn new(id: Id) -> Self {
-        Self { id }
+    /// Construct new [`Definition`]
+    #[inline]
+    pub fn new(id: <Definition as Identifiable>::Id) -> Self {
```

Comment:
Could be `const`?

Reply:
Unfortunately no. `BTreeMap::new()` is not yet stable as `const` as my copiler tells me